manifests/server/config.pp

```diff
@@ -179,7 +179,7 @@
   if $handle_creds {
     file { $rcfile:
       ensure  => file,
-      content => template('mongodb/mongorc.js.erb'),
+      content => template('mongodb/mongoshrc.js.erb'),
```

Comment:
Isn't this really client side config that should be managed in client.pp or client/config.pp?

Reply:
Technically this isn't configuration, it is a set of commands that get executed by the client when loading this script. It is required by the provider to set up the server correctly, not configuring the client. Apart from a package install the client doesn't need configuration. So I would leave it in the server/config.pp.
It contains for example the credentials of the admin user, which obviously are specific to that particular server.

Reply:
We don't have to do anything about this now, as it's the same with mongorc.
But IMO users should be able to manage this file however they like without breaking the provider.

Reply:
Agreed, I created an issue for this: https://github.com/voxpupuli/puppet-mongodb/issues/717